### PR TITLE
Updated openscap default content

### PR DESCRIPTION
**Pull request by shlomizadok**
Update to how default SCAP content is achieved

### CODE REVIEW

File: plugins/foreman_openscap/0.5/index.md [hunk 1]
```diff
@@ -124,9 +124,28 @@ which runs the SCAP client at the schedule select when creating the policy.
...
+
+Instead of auto-generating default SCAP content when installing foreman_openscap, you can now accomplish that with a rake task.
+
+__Creating default SCAP content__
+
+* Install smart_proxy_openscap on one (or more) of your proxies
+* Refresh features of that proxy (so it will register with OpenSCAP feature on the Foreman)
+* from terminal run `foreman-rake foreman_openscap:bulk_upload:default`
+
+This will search for scap-security-guide SCAP contents an create SCAP content on the Foreman.
```
Comment (ares) [inline plugins/foreman_openscap/0.5/index.md]: `s/an/and/`
also maybe this step could be descried a bit more, e.g. mention that Foreman will upload the file to the proxy which will parse it, what proxy is being used if you have more than one (I guess first wins)

**Comment by ares** [comment]
Could you also please mention somewhere trusted_hosts authentication discussed at https://github.com/theforeman/smart_proxy_openscap/pull/25#discussion_r56816506
After that I'm happy to merge that smart_proxy PR.

Otherwise I think that the text is fine. I would leave spelling and grammar for someone else since I'm not native speaker.

**Comment by shlomizadok** [comment]
@ares - I will but I think this belongs to another PR (which I will create as soon as the theforeman/smart_proxy_openscap#25 is merged - at it requires more documentation regarding how upgrading from 0.4.x to 0.5.x

**Comment by ares** [comment]
ok, if no one will volunteer for the grammar check in few days I'll merge it

**Comment by ares** [comment]
Thanks @shlomizadok, merging!

**Pull request merged**